### ISSUE TITLE
Dataset inconsistencies

### ISSUE THREAD

**Issue by lennartmoritz** [stormyoung/PRAI-1581#1]
_Note: I've downloaded the Dataset via Google Drive. I couldn't figure the Baidu Disk download out (I don't speak the language)._

I'm trying to work with your dataset and want to recreate your split of test images and training images. But there are **inconsitencies between the data in your partitions.pkl file, the actual images in your images folder and the statements you make in your paper** "Person Re-identiﬁcation in Aerial Imagery" in the 12 Apr 2020 version. For more details about inconsistencies read my last paragraph. 
Can you please update inconsistent parts to make your data comparable for other research?

But I also need to know **how you have split your test images in gallery and query sets**. This is important to be able to compare my results to yours in the end. 
Can you please update your partitions file with that information?

Your paper states:
> Our PRAI-1581 dataset contains a total of 39,461 images for 1581 individual identities. We randomly divide the dataset into training and test set for only once and keep the partition ﬁxed in all the experiments for fair comparison. The ratio of training and testing set is roughly set to 1:1. The training set includes 19,523 images of 782 identities. And the rest of the dataset is used as the test set with 19,938 images of 799 identities. To be more speciﬁc, the numbers of query and gallery images are 4680 and 15,258 respectively.

### Inconsistencies:
- Actual # of Images in the dataset is 39,455. But should be 39,461 according to the paper.
- The # of training images according to partitions.pkl is 20,658. But should be 19,523 according to the paper.
- The # of test images according to partitions.pkl is 18803. But should be 19,938 according to the paper.
- The # of identities in the training set according to partitions.pkl is 818. But should be 782 according to the paper.

**Comment by stormyoung** [comment]
Thank you for your question and careful check. 
We have update the PRAI-1581 dataset consistented with our paper and you can download it by the updated links of Baidu Disk or Google drive in README.md.
Please be free to contact with us if you have any questions.
